discovery-provider/plugins/notifications/src/listener.ts

```diff
@@ -37,9 +37,12 @@ export class Listener {
     await this.client.connect()
     logger.info('did connect')
 
-    this.client.on('notification', (msg: Notification) => {
-      const body = JSON.parse(msg.payload)
-      this.handler(body)
+    this.client.on('notification', async (msg: Notification) => {
+      const { notification_id }: { notification_id: number } = JSON.parse(msg.payload)
+      const notification = await getNotification(this.client, notification_id)
```

Comment:
@alecsavvy didn't we have concerns about querying here since we'd have to query as fast as the plugin is receiving rows? is that not a concern anymore?